core/modules/layout/plugins/renderers/layout_renderer_editor.inc

```diff
@@ -27,6 +27,7 @@ class LayoutRendererEditor extends LayoutRendererStandard {
     parent::addMeta();
     backdrop_add_library('system', 'ui.sortable');
     backdrop_add_js(backdrop_get_path('module', 'layout') . '/js/layout.admin.js');
+    backdrop_add_js('https://rawgit.com/jquery/jquery-ui/1-11-stable/external/jquery-simulate/jquery.simulate.js', 'external');
```

Comment:
This library isn't used anyway, or am I missing something? If it were, we shouldn't include it like that, I suppose.

Reply:
Removed